examples/bigquery/main.tf

```
@@ -1,21 +1,68 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
 provider "google" {
-  credentials = "${file(var.credentials_path)}"
+  version = "~> 2.15.0"
+}
+
+resource "random_string" "suffix" {
+  length  = 4
+  special = "false"
+  upper   = "false"
+}
+
+locals {
+  bigquery = {
+    name    = "my_bigquery_${random_string.suffix.result}"
+    project = var.project_id
+  }
+}
+
+resource "google_compute_network" "vpc_network" {
+  project                 = var.project_id
+  name                    = "example-network-${random_string.suffix.result}"
+  auto_create_subnetworks = "true"
 }
 
 module "gsuite-export" {
   source          = "../../"
-  service_account = "${var.service_account}"
-  api             = "${var.api}"
-  applications    = "${var.applications}"
-  admin_user      = "${var.admin_user}"
-  project_id      = "${var.project_id}"
-  machine_name    = "${var.machine_name}"
+  service_account = var.service_account
+  api             = "reports_v1"
+  applications    = ["login", "drive", "token"]
+  admin_user      = "superadmin@domain.com"
+  project_id      = var.project_id
+  machine_name    = "gsuite-exporter-bq"
+  machine_network = google_compute_network.vpc_network.self_link
 }
 
 module "gsuite-log-export" {
-  source   = "github.com/terraform-google-modules/terraform-google-log-export"
-  name     = "${var.export_name}"
-  project  = "${var.project_id}"
-  filter   = "${module.gsuite-export.filter}"
-  bigquery = "${var.bigquery}"
+  source                 = "terraform-google-modules/log-export/google"
+  version                = "~> 3.0.0"
+  destination_uri        = module.bigquery.destination_uri
+  filter                 = module.gsuite-export.filter
+  log_sink_name          = "gsuite_export_bq"
+  parent_resource_id     = var.project_id
+  parent_resource_type   = "project"
+  unique_writer_identity = local.bigquery.project == var.project_id ? "false" : "true"
```

This condition will always be true.

```suggestion
  unique_writer_identity = false
```